spaceapi/src/lib.rs

```diff
@@ -44,10 +44,12 @@
 //!     let stringstatus = jsonstatus.to_string();
 //!     # }
 
+#[macro_use]
+extern crate log;
```

Comment:
You **have** to put it in the top level module of a crate, otherwise it won't work.